kotlin-checks-test-sources/src/main/kotlin/checks/FinalFlowOperationCheckSample.kt

```diff
@@ -0,0 +1,26 @@
+package checks
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.count
+import kotlinx.coroutines.flow.filter
+import kotlinx.coroutines.flow.map
+
+class FinalFlowOperationCheckSample {
+    suspend fun foo(flow: Flow<String>) {
```

Comment:
I think it will be nice to add a few more cases, where `Flow` is returned as a result of a lambda or passed as an argument, or returned from a function, etc. They should be handled by the implementation, but still nice to double-check.

Reply:
WDYT of these new ones?

Reply:
Looks good, I've just realized you're not checking the message and precise issue location.